nixos/modules/virtualisation/containers.nix

```
@@ -48,6 +48,12 @@ in
       description = "containers.conf configuration";
     };
 
+    storageConf.settings = mkOption {
```

```suggestion
    storage.settings = mkOption {
```
To align this with `registries`. `containersConf` was used so the option wasn't `virtualisation.containers.containers.settings`